flocker/node/agents/ebs.py

```diff
@@ -393,6 +375,10 @@ def _next_device(self, instance_id):
             file_name = u'/dev/sd' + suffix
             if file_name not in devices:
                 return file_name
+
+        # Could not find any suitable device that is available
+        # for attachment. Log to Eliot before giving up.
+        NO_AVAILABLE_DEVICE(devices=devices).write()
```

Comment:
I suppose this'll be redundant if / when we implement https://clusterhq.atlassian.net/browse/FLOC-1887 "Allow users to know when they run out of device names to attach EBS volume" and the suggested new ``CannotAttach`` exception proposed in that issue.

Perhaps we should just fix that issue instead.

However, the log message is useful in the meantime and it'd be even more useful (I think) if it included the blockdevice_id and the real allocated device path...the one found by _wait_for_new_device (above) which is different, right?